README.md

```diff
@@ -68,28 +68,53 @@ Get the repository using `go get`:
 ```bash
 go get github.com/codilime/floodgate
 ```
-In its current development state Floodgate checks if a `config.yaml` is present in the root directory.
-Navigate to the directory:
+This will download, compile and install Floodgate to `$GOPATH/bin`. If this directory is present in your `$PATH`, the `floodgate` command will be available from any directory.
+
+Create a `config.yaml` file. An example one can be found in `examples/config.yaml`.
+Floodgate checks if a config file is present in:
 ```bash
-cd $GOROOT/github.com/codilime/floodgate
+$HOME/.config/floodgate/config.yaml
 ```
-Create a `config.yaml` file. An example config can be found in `example.yaml`.
-Additionally, you can find example resources in the `resources` directory.
+You can also specify a path to the config file using the `--config` flag.
+
+Additionally, you can find example resources in the `examples/resources` directory.
 
-Execute `go build` and run the binary:
+**Note:** To allow Floodgate to control pipeline templates apply those commands to your environment:
 ```bash
-go build && ./floodgate
+hal config features edit --pipeline-templates true
+hal config features edit --managed-pipeline-templates-v2-ui true
+hal deploy apply
 ```
-You can also use `go install` so that the `floodgate` command is available from any directory.
 
 To run tests use:
 ```bash
-go test ./...
+go test $GOPATH/src/github.com/codilime/floodgate/...
```

Comment:
I don't think GOPATH is important if you're using go modules, which we support. Correct me if I'm wrong, though :)

Reply:
Well, even now `go get` has to save the module (and its dependencies) somewhere (`$GOPATH/src`). If no gopath is set, the default location is `$HOME/go/src`. So maybe I should include that information?

Reply:
I just think that most people would run tests while in the directory, not from somewhere else, so the way it was (`./...`) should be enough

Reply:
Done